samples/kfp-user-guide/README.md

```diff
@@ -0,0 +1,275 @@
+# ml-pipeline-persistenceagent-bb9986b46-l4kcx                     1/1     Running   0          8d
+# ml-pipeline-scheduledworkflow-b959d6fd-bnc2w                     1/1     Running   0          33d
+# ml-pipeline-ui-6f68595ff-tw295                                   1/1     Running   0          8d
+# ml-pipeline-viewer-controller-deployment-7f65754d48-xz6jh        1/1     Running   0          33d
+# ml-pipeline-viewer-crd-7bb858bb59-k9mz9                          1/1     Running   0          33d
+# ml-pipeline-visualizationserver-d558b855-tdbwc                   1/1     Running   0          33d
+```
+
+Then, we need to find the pipeline ID that we want to execute.
+```shell
+kfp pipeline list
+# +--------------------------------------+-----------------------------------+---------------------------+
+# | Pipeline ID                          | Name                              | Uploaded at               |
+# +======================================+===================================+===========================+
+# | 925415d5-18e9-4e08-b57f-3b06e3e54648 | echo_pipeline                     | 2020-07-02T17:55:30+00:00 |
+# +--------------------------------------+-----------------------------------+---------------------------+
+```
+
+Next, submit the pipeline ID for execution
+```shell
+kfp run submit kfp run submit -e experiment-name -r run-name -p 925415d5-18e9-4e08-b57f-3b06e3e54648
+# Creating experiment experiment-name.
+# Run bb96363f-ec0d-4e5a-9ce9-f69a485c2d94 is submitted
+# +--------------------------------------+----------+----------+---------------------------+
+# | run id                               | name     | status   | created at                |
+# +======================================+==========+==========+===========================+
+# | bb96363f-ec0d-4e5a-9ce9-f69a485c2d94 | run-name |          | 2020-07-02T19:08:58+00:00 |
+# +--------------------------------------+----------+----------+---------------------------+
+```
+
+Lastly, we can check the status of the pipeline execution
+```shell
+kfp run get bb96363f-ec0d-4e5a-9ce9-f69a485c2d94
+# +--------------------------------------+----------+-----------+---------------------------+
+# | run id                               | name     | status    | created at                |
+# +======================================+==========+===========+===========================+
+# | bb96363f-ec0d-4e5a-9ce9-f69a485c2d94 | run-name | Succeeded | 2020-07-02T19:08:58+00:00 |
+# +--------------------------------------+----------+-----------+---------------------------+
+```
+
+
+### 4. Tekton Exclusive: Execute Tekton pipelines without using Kubeflow pipeline
+For all the compiled Kubeflow pipeline, we can extract the file into `.yaml` format and run it directly on Kubernetes using the Tekton CRD. However, this method should **only use for Tekton development** because Kubeflow pipeline cannot track these pipelines that run directly on Tekton.
```

Comment:
```suggestion
For all the compiled Kubeflow pipelines, we can extract them into `.yaml` format and run them directly on Kubernetes using the Tekton CRD. However, this method should **only use for Tekton development** because Kubeflow Pipelines engine cannot track these pipelines that run directly on Tekton.
```